benchmarks/unordered-containers-benchmarks.cabal

```diff
@@ -23,7 +23,7 @@ executable unordered-containers-benchmarks
     base,
     bytestring,
     containers,
-    criterion,
+    gauge,
```

Comment:
Not sure whether this file can be removed now that the dependency cycle has been removed?